### ISSUE TITLE
Could you release your metric calculation script please?

### ISSUE THREAD

**Issue by HWalkingMan** [zsyOAOA/DifFace#4]
Your work is awesome! I have tested with your pre-trained model on CelebaTest and got amazing visual results.

However, I noticed that your paper and the VQFR's paper both provide metrics for testing VQFR on CelebaTest, and these metrics are different.

Thus, I used your model to infer on the CelebaTest dataset provided by VQFR (link here)[https://drive.google.com/drive/folders/1CPdaCijkhl8ojg9ZDCPXntQrnw-8tUeD?usp=sharing] and use the calculation script provided by VQFR(link here)[https://github.com/TencentARC/VQFR/tree/main/metric_paper], the unexpected results were obtained.

Therefore, I am very curious about the quantitative metrics mentioned in the paper. How do you calculate the metrics? Could you release your metric calculation script please?

**Comment by zsyOAOA** [comment]
For PSNR and LPIPS: https://github.com/chaofengc/IQA-PyTorch

For LPIPS (VGG): https://github.com/richzhang/PerceptualSimilarity

For IDS: see the script of VQFR.

For FID: https://github.com/mseitzer/pytorch-fid

As for as I know, for the metric of FID, VQFR and GFPGAN are calculated between the restored faces and the whole faces in FFHQ. However, in our paper, it is directly calculated between the restored faces and the corresponding ground truth.

**Comment by HWalkingMan** [comment]
Thanks for your scripts~
